.github/workflows/ROMROM-IOS-TESTFLIGHT.yaml

```
@@ -263,17 +263,47 @@ jobs:
+              begin
+                pilot(
+                  api_key: api_key,
+                  app_identifier: "com.alom.romrom",
+                  distribute_only: true,
+                  groups: ["alom"],
+                  notify_external_testers: false
+                )
+                puts "✅ alom 그룹 배포 성공!"
+              rescue => e
+                puts "⚠️ alom 그룹 배포 실패: #{e.message}"
+                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)